0085-Hex-Coverage-Limit.md

```diff
@@ -0,0 +1,91 @@
+|   D   |   -93.60 dBm  |12/05/2022 11:51:01          | Fail  | 16 (16 * 1)               | 0   (16 * 0)       |
+|   E   |   -94.69 dBm  |08/01/2022 05:01:59          | Fail  | 16 (16 * 1)               | 0   (16 * 0)       |
+
+**Table Key:**
+- **Radio:** Example radio name.
+- **Signal Strength:** The Signal Strength of the res12 hex from the modeled coverage explorer.
+- **Heartbeat:** The date/time that the Heartbeat Streak started.
+- **Rank:** The assigned rank based on which radio has the strongest Signal Strength (or Heartbeat Streak if tied for Signal Strength).
+- **Coverage Points Per HIP 74:** The amount of modeled coverage points currently awarded under HIP 74.
+- **New Coverage Points:** The amount of modeled coverage points that would be awarded under this HIP.
+
+**Table Explanation:**
+
+Since Radio A has the highest signal strength in that hex, it will be ranked as "1", and granted a 1X multiplier, which will award it with the full 16 (16 x 1) modeled coverage points for that epoch.
+
+Since radios B and C tied in Signal Strength, the Heartbeat Streak date is used to determine which radio is ranked next. For this example, radio B had its Heartbeat Streak start on 12/01/2022 01:01:01 while radio C had a its HeartBeat streak start on 12/02/2022 12:11:01. Therefore, since radio B has an earlier Heartbeat Streak, it will be ranked "2", while radio C having a rank of "Fail".
+
+Since radio C, D and E had the lowest signal strength out of all five (5) radios, and only the top two (2) radios will earn PoC rewards, radios C, D and E will not earn any modeled coverage points, and are ranked as "Fail".
+
+As noted in the example above, outdoor radio deployers will now need to be cognizant of where they are placing their radios in order to maximize modeled coverage point. 
+
+## Drawbacks:
+Implementing this proposal could increase the complexity of modeled coverage scores due to adding additional variables used to calculate total MOBILE rewards.
+
+Additionally, radio deployers may lose out on awarded coverage points in instances where multiple radios are set up in the same hex.
+
+## Rationale and Alternatives:
+An alternative would be allowing radios and hexes to keep earning the defined amount of modeled coverage points as described in HIP 74. However, this may prevent or stagnate the network's growth because this method incentivizes placement of radios to minimize overlapping coverage, which is misaligned with expanding urban coverage needs.
+
+## Unresolved Question:
+1. Should MOBILE Mapping Signal Strength be used in combination with Modeled Signal Strength?
+
+## Deployment Impact:
+HIP 85 affects only Outdoor radios, and coverage from indoor radios will continue to earn Modeled Coverage Points based on HIP 74. New fields will need to be added into the Modeled Coverage Explorer to make Radio Hex Ranks and Heartbeat Streaks visible.
+
```

Comment:
@heatedlime can you speak to the impact on any smart contracts? Are oracles just affected here or does there need to be some change to on-chain contracts?

Reply:
Leaving the implementation open to interpretation by Helium Core Developers.

Reply:
Next time you have to edit the file, but I'll add this in for you this time :).